crates/typst-library/src/text/mod.rs

```diff
@@ -1485,4 +1556,36 @@ mod tests {
     fn test_text_elem_size() {
         assert_eq!(std::mem::size_of::<TextElem>(), std::mem::size_of::<EcoString>());
     }
+
+    #[test]
+    fn test_text_tag_parsing() {
```

Comment:
why test this from Rust? as it does not test the error message, we need to test pretty much everything from Typst anyways and then it seems redundant.